ios/FluentUI/List/ListItem.swift

```diff
@@ -0,0 +1,209 @@
+                            .frame(minHeight: ListItemTokenSet.subtitleTwoLineHeight)
+                    } else if layoutType == .threeLines {
+                        subtitleView
+                            .font(Font(tokenSet[.subtitleThreeLinesFont].uiFont))
+                            .frame(minHeight: ListItemTokenSet.subtitleThreeLineHeight)
+                        Text(footer)
+                            .foregroundColor(Color(uiColor: tokenSet[.footerColor].uiColor))
+                            .font(Font(tokenSet[.footerFont].uiFont))
+                            .frame(minHeight: ListItemTokenSet.footerHeight)
+                            .lineLimit(state.footerLineLimit)
+                            .truncationMode(state.footerTruncationMode)
+                    }
+                }
+            }
+        }
+
+        @ViewBuilder
+        var accessoryView: some View {
+            if let accessoryType = state.accessoryType,
+               let icon = accessoryType.icon,
+               let iconColor = accessoryType.iconColor(tokenSet: tokenSet, fluentTheme: fluentTheme) {
+                let image = Image(uiImage: icon)
+                    .foregroundColor(Color(uiColor: iconColor))
+                if accessoryType == .detailButton {
+                    SwiftUI.Button {
+                        if let onAccessoryTapped = state.onAccessoryTapped {
+                            onAccessoryTapped()
+                        }
+                    } label: {
+                        image
+                    }
+                } else {
+                    image
+                }
+            }
+        }
+
+        @ViewBuilder
+        var backgroundView: some View {
+            if let backgroundColor = state.backgroundStyleType.defaultColor(tokenSet: tokenSet) {
+                Color(backgroundColor)
+            }
+        }
+
+        @ViewBuilder
+        var contentView: some View {
+            HStack(alignment: .center) {
+                leadingView
+                    .padding(.trailing, ListItemTokenSet.horizontalSpacing)
+                labelStack
+                    .padding(.trailing, ListItemTokenSet.horizontalSpacing)
+                Spacer()
+                trailingView
+                    .tint(Color(fluentTheme.color(.brandForeground1)))
+                accessoryView
+                    .padding(.leading, ListItemTokenSet.horizontalSpacing)
+            }
+            .padding(EdgeInsets(top: ListItemTokenSet.paddingVertical + 1,
+                                leading: ListItemTokenSet.paddingLeading,
+                                bottom: ListItemTokenSet.paddingVertical + 1,
+                                trailing: ListItemTokenSet.paddingTrailing))
+            .frame(minHeight: layoutType.minHeight)
+            .background(backgroundView)
+            .listRowInsets(EdgeInsets())
+        }
+
+        return contentView
+    }
+
+    var state: ListItemState = ListItemState()
```

Comment:
This will get recreated on every view re-init, which can happen at any time. SwiftUI destroys and rebuilds views whenever it wants. Should it be a @StateObject ?